src/test_workflow/benchmark_test/benchmark_args.py

```diff
@@ -69,20 +70,23 @@ def __init__(self) -> None:
                             help="A comma-separated list of key=value pairs that will be added to jvm.options as JVM system properties.")
         parser.add_argument("--additional-config", nargs='*', action=JsonArgs, dest="additional_config",
                             help="Additional opensearch.yml config parameters passed as JSON")
+        parser.add_argument("--use-50-percent-heap", dest="use_50_percent_heap", action="store_true",
+                            help="Use 50 percent of physical memory as heap.")
         parser.add_argument("--ml-node-storage", dest="ml_node_storage",
                             help="User provided ml-node ebs block storage size defaults to 100Gb")
         parser.add_argument("--data-node-storage", dest="data_node_storage",
                             help="User provided data-node ebs block storage size, defaults to 100Gb")
-        parser.add_argument("--workload", dest="workload", help="workload type for the OpenSearch benchmarking",
-                            required=True)
+        parser.add_argument("--workload", dest="workload", required=True,
+                            help="workload type for the OpenSearch benchmarking")
         parser.add_argument("--benchmark-config", dest="benchmark_config",
                             help="absolute filepath to custom opensearch-benchmark.ini config")
```

Comment:
The help section says `opensearch-benchmark.ini` but it should be `benchmark.ini`.
```
hoangia@3c22fbd0d988 Desktop % ls -lrt ~/.benchmark
drwxr-xr-x  6 hoangia  staff   192 Jun 10  2022 benchmarks
drwxr-xr-x  3 hoangia  staff    96 Aug  9  2022 logs
-rw-r--r--  1 hoangia  staff   654 Mar 14 11:10 benchmark-in-memory-it.ini
-rw-r--r--  1 hoangia  staff   768 Mar 14 11:10 benchmark-os-it.ini
-rw-r--r--  1 hoangia  staff   757 Mar 27 11:54 benchmark.ini
```

Reply:
Thanks, will correct this.